docker/container.go

```diff
@@ -128,6 +131,21 @@ func (cd *ContainerDetails) GetName() string {
 	panic("Unsupported type")
 }
 
+// GetPorts gets the ports to open for the node, if instructed.
+func (cd *ContainerDetails) GetPorts() string {
+	switch cd.Type {
+	case Node:
+		if cd.Resources.Ports != nil && len(cd.Resources.Ports) > cd.Node {
+			return cd.Resources.Ports[cd.Node]
```

Comment:
I don't believe this is the intended behavior. It should most likely return all of the ports that the node is in possession of.

Reply:
This exposes ports outside the docker network to the host. It's a proxy binding between a port inside the container and a port on the host. Does that make sense?

Reply:
I am talking about something super tiny. It should most likely return  cd.Resources.Ports and then in the run command function, use a for loop to add a -p <port> for each object. In short, each node has a cd.Resources object, which may contain an arbitrary amount of ports. Does that make sense?